test/e2e/framework/pod/wait.go

```diff
@@ -34,6 +34,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/kubectl/pkg/util/podutils"
+	"k8s.io/kubernetes/test/e2e/framework"
```

Comment:
@jingxu97 looks like you will need a `./hack/update-bazel.sh` to include new dependency